edb/tools/test/__init__.py

```diff
@@ -80,8 +83,13 @@
               help='package name to measure code coverage for, '
                    'can be specified multiple times '
                    '(e.g --cov edb.common --cov edb.server)')
-def test(*, files, jobs, include, exclude, verbose, quiet, debug,
-         output_format, warnings, failfast, shuffle, cov, repeat):
+@click.option('--time-file', type=click.File('a+'), metavar='FILEPATH',
```

Comment:
I'd call this `--log-running-times`

Reply:
Or, rather `--running-times-log`

Reply:
Oh actually it also reads this file for calculating distribution - should we create a separate option for reading?

Reply:
Why a+?

Reply:
It's both for reading and writing, and I'd like the file to be created if not exist. `r+` doesn't create the file, and `w+` overwrites the file on open.